lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var _ = require('lodash');
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  // variables should be defined here
+  var indentMode = 0;
+
+  //--------------------------------------------------------------------------
+  // Helpers
+  //--------------------------------------------------------------------------
+
+  function getNodeIndent(node, byLastLine, excludeCommas, indentType) {
```

Comment:
This is taken from [here](https://github.com/eslint/eslint/blob/13c9e989c063321bfe1a1feb61e90b91adc2cee4/lib/rules/indent.js#L168) right? Please put this into a separate module and add a comment stating where you got the code from.